File: 0601_0700/0693/0693.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    p = {i:0 for i in "abcdefghijklmnopqrstuvwxyz"}
    s, t = f.readline().split()
    r = (len(s) == len(t))
    if r:
        for i in range(len(s)):
            p[s[i].lower()] += 1
            p[t[i].lower()] -= 1
        r = all(x == 0 for x in p.values())
    q.write("Yes" if r else "No")

File: 0601_0700/0693/0693.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int p[26] = {0}, r;
	char s[51], t[51];

	fscanf(f, "%s%s", s, t);
	if ((r = (strlen(s) == strlen(t))))
	{
		for (int i = 0; s[i]; ++i)
			++p[tolower(s[i]) - 'a'], --p[tolower(t[i]) - 'a'];
		for (int i = 0; r && i < 26; ++i)
			r = !p[i];
	}
	fprintf(q, r ? "Yes" : "No");
	return 0;
}
